package-lock.json

```diff
@@ -17655,8 +17655,8 @@
       "integrity": "sha512-6qE4B9deFBIa9YSpOc9O0Sgc43zTeVYbgDT5veRKSlB2+ZuHNoVVxA1L/ckMUayV9Ay9y7Z/SZCLcGteW9i7bg=="
     },
     "mattermost-redux": {
-      "version": "github:mattermost/mattermost-redux#6fdb4667fdb689ac209eb27d2726922d8b422104",
-      "from": "github:mattermost/mattermost-redux#6fdb4667fdb689ac209eb27d2726922d8b422104",
+      "version": "github:mattermost/mattermost-redux#7a241e2a4f56217320ae79facca5a3a3fb83e95d",
```

Comment:
Did you mean to update `package.json` as well?

Reply:
Nope!